src/Symfony/Component/Debug/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.3.0
+-----
+
+* made the `ErrorHandler` and `ExceptionHandler` classes final
```

Comment:
They aren't actually final, just marked as such. It would make more sense in my opinion, to write "marked the ..."

Reply:
Yeah I thought about this but I actually searched how these kind of deprecations were handled in the past in the changelogs, and they were done like that 😕

Should I add also add an entry for the 5.0.0 with the fact that these classes will really be final at this point ?

Reply:
`@final since Symfony 4.3` so the changelog is correct. Both `final class` and `@final` are final :)